test/integration/publish_int_test.go

```diff
@@ -378,29 +406,24 @@ authors:
 
 					cp.Expect("Y/n")
 
-					snapshot := cp.Snapshot()
```

Comment:
I removed this because with the dependencies from the first test the snapshot was too small to capture all of the YAML output.

Reply:
Still, the test was useful, right? Can we not add some sort of `cp.Expect()` or `cp.ExpectRe()` call that would be equivalent or close to it?

Reply:
Sure, it's just frail as the correct YAML formatting has to be contained within the snapshot. 

I've added the parsing back and a conditional to attempt parsing the snapshot or not. If there is an error parsing the YAML we will also fall back to the provided ingredient name, namespace, and version.